lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2902,6 +2970,50 @@ void SplitEmitter::emitModule(EmittedModule &mod) {
 // MLIRModuleEmitter
 //===----------------------------------------------------------------------===//
 
+namespace {
+/// Commandline parser for VerilogEmitterOptions.  Delegates to the parser
+/// defined by VerilogEmitterOptions.
+struct VerilogEmitterOptionsParser
```

Comment:
I'd really prefer for all this to be in a .cpp file in lib/Support instead of in the verilog printer :-)